css/style.css

```diff
@@ -34,15 +43,19 @@ header {
 .main-page-h1 {
   font-size: 36px;
   font-weight: 700;
+  padding: 0;
+  margin-bottom: 0;
+  height: 250px;
 }
```

Comment:
- [ ] Adding a fixed height to `.main-page-h1` causes an overlap of elements as seen below.

![Screenshot 2021-11-25 at 12-49-54 Portfolio](https://user-images.githubusercontent.com/4247740/143418855-7d559c64-02f1-4d4d-aa07-8efa744b14c2.png)

### Suggestion

- [ ] I suggest that you remove the `height` and instead add a `line-height` to the `<h1>` element

Reply:
I deleted 'height' and add 'line-height: 54px;'